resources/public/api/conf/2.0/application.raml

```diff
@@ -72,6 +72,14 @@ documentation:
                   example: 4557ecb5-fd32-48cc-81f5-e6acd1099f3c
               get: !include uk_property_period_summary_retrieve.raml
               put: !include uk_property_period_summary_amend.raml
+    /non-furnished-holiday-lettings:
+      /{nino}:
+        uriParameters:
+          nino:
+            description: National Insurance number, in the format AA999999A.
+            type: string
+            example: TC663795B
+        post: !include uk_property_non_fhl_period_summary_create.raml
```

Comment:
All of the historic endpoints have been moved together (so that a group heading appears in the documentation for fhl and non-fhl historic). 

All of this section except the "post..." line will need deleting and the "post..." line should be moved to what will be line 106 (I think), immediately before the "/non-furnished-holiday-lettings" "/{taxYear}" line.